packages/ui-components/src/components/select-option/select-option.tsx

```diff
@@ -95,11 +98,25 @@ export class KvSelectOption implements ISelectOption, ISelectOptionEvents {
 							</div>
 						)}
 						<div class="text-container">
-							<div class="item-label" part="label">
-								{this.isDirty && <kv-dirty-dot />}
-								{this.label}
+							<div class="left-content">
+								<div class="item-label" part="label">
+									{this.isDirty && <kv-dirty-dot />}
+									{this.label}
+								</div>
+							</div>
+							<div class="right-content">
+								{this.description && <div class="item-description">{this.description}</div>}
+								{this.action && (
+									<div class="action-container">
+										<kv-action-button-icon
+											type={EActionButtonType.Ghost}
+											icon={this.action.icon}
+											onClick={event => event.stopPropagation()}
```

Comment:
These onClick is necessary to avoid to close the dropdown?

Reply:
Afterall, seem that this is really necessary otherwise the dropdown will think that the option was selected.